builtin/mainmenu/tab_online.lua

```diff
@@ -174,12 +174,12 @@ local function get_formspec(tabview, name, tabdata)
 				"server_view_clients.png") .. ";btn_view_clients;]"
 		end
 
-		if selected_server.url then
-			retval = retval .. "tooltip[btn_server_url;" .. fgettext("Server Website") .. "]"
-			retval = retval .. "style[btn_server_url;padding=6]"
-			retval = retval .. "image_button[" .. (can_view_clients_list and "4" or "4.5") .. ",1.3;0.5,0.5;" ..
-			core.formspec_escape(defaulttexturedir .. "server_url.png") .. ";btn_server_url;]"
-		end
+			if selected_server.url then
+				retval = retval .. "tooltip[btn_server_url;" .. fgettext("Server Website") .. "]"
+				retval = retval .. "style[btn_server_url;padding=6]"
+				retval = retval .. "image_button[" .. (can_view_clients_list and "4" or "4.5") .. ",1.3;0.5,0.5;" ..
+				core.formspec_escape(defaulttexturedir .. "server_url.png") .. ";btn_server_url;]"
```

Comment:
Sorry, I think you misunderstood me. I wanted just this line (the "continuation of the expression") to be indented by one or two tabs, not the entire `if` statement.

Reply:
Should be good now?